webpack.config.js

```diff
@@ -44,17 +45,20 @@ module.exports = {
 			{
 				test: /\.json/,
 				loader: 'json',
-			}
-		]
+			},
+		],
 	},
 	resolve: {
-		extensions: ['', '.js', '.jsx']
+		extensions: ['', '.js', '.jsx'],
 	},
 	plugins: [
 		// Move static images over as well. This isn't normally something you do
 		// with webpack but it works fine for our case.
 		new CopyWebpackPlugin([
-			{ from: './src/docs/img', to: 'img' }
-		])
-	]
+			{ from: './src/docs/img', to: 'img' },
+		]),
+		new webpack.DefinePlugin({
+			PRODUCTION : process.env.NODE_ENV === 'production',
```

Comment:
I just called this `PRODUCTION` because it felt weird for client side code to be looking at a value called `NODE_ENV`, but I'm cool with changing it, if anyone feels strongly.